test/timeout.go

```diff
@@ -0,0 +1,28 @@
+package test
+
+import (
+	"os"
+	"runtime/pprof"
+	"testing"
+	"time"
+)
+
+const testTimeout = 20 * time.Second
+
+// Timeout implements a test level timeout.
+func Timeout(t *testing.T) func() {
```

Comment:
you are passing t but is not being used.

Reply:
Fixed.